README.md

```
@@ -18,45 +20,57 @@ This repository makes Nightly builds for **x86_64** (Standard), **x86_64_v3** (C
 # READ THIS IF YOU HAVE ISSUES
 
-If you are on wayland (specially GNOME wayland) and get freezes or crashes, you are likely affected by this issue that affects all Qt6 apps: [citron-neo#50](https://github.com/citron-neo/CI/pull/50)
+If you are on wayland (specially GNOME wayland) and get freezes or crashes, you are likely affected by this issue that affects all Qt6 apps: https://github.com/citron-neo/CI/issues/50
 
 To fix it simply set the env variable `QT_QPA_PLATFORM=xcb`
 
-**Also, are you looking for AppImages of other emulators? Check:** [AnyLinux-AppImages](https://pkgforge-dev.github.io/Anylinux-AppImages/)
+**Also, are you looking for AppImages of other emulators? Check:** [AnyLinux-AppImages](https://pkgforge-dev.github.io/Anylinux-AppImages/) 
 
----
+----
```

markdownlint-cli2 (0.22.1)

[warning] Horizontal rule style
Expected: ---; Actual: ----

(MD035, hr-style)